### PR TITLE
Fix files finding

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ✔️
| Tests pass?   | ✔️
| Depends on  | https://github.com/yiisoft/yii-debug/pull/61

:warning:  Before merge need to check in real integration
- [x] Integration checked

### CODE REVIEW

**Comment by samdark** [comment]
What problem is being fixed?

File: src/Repository/CollectorRepository.php [hunk 1]
```diff
@@ -69,9 +70,9 @@ private function loadData(string $fileSuffix, ?string $id = null): array
         }
 
         $data = [];
-        $dataFiles = \glob($this->path . '/yii-debug*' . $fileSuffix, GLOB_NOSORT);
+        $dataFiles = \glob($this->path . '/**/**/' . $fileType, GLOB_NOSORT);
         foreach ($dataFiles as $file) {
-            $id = \basename($file, $fileSuffix);
+            $id = \basename($file, $fileType);
```
Comment (rustamwin) [inline src/Repository/CollectorRepository.php:75]:
```suggestion
            $dir = \dirname($file);
            $id = \substr($dir, \strlen($this->path));
```

File: src/Repository/CollectorRepository.php [hunk 2]
```diff
@@ -56,11 +56,12 @@ public function getDumpObject(string $id, ?string $collector): array
...
 
-    private function loadData(string $fileSuffix, ?string $id = null): array
+    private function loadData(string $fileType, ?string $id = null): array
     {
         clearstatcache();
         if (!empty($id)) {
-            $file = $this->path . DIRECTORY_SEPARATOR . $id . $fileSuffix;
+            $files = \glob($this->path . '/**/' . $id . '/' . $fileType, GLOB_NOSORT);
+            $file = current($files);
             if (!file_exists($file) || is_dir($file)) {
```
Comment (rustamwin) [inline src/Repository/CollectorRepository.php:65]:
```suggestion
            if ($file === false || !file_exists($file) || is_dir($file)) {
```

**Review by rustamwin — CHANGES_REQUESTED** [review]

**Comment by xepozz** [comment]
> What problem is being fixed?

[This PR](https://github.com/yiisoft/yii-debug/pull/61) was changed directory hierarchy and now API won't find any debug-files.

**Comment by xepozz** [comment]
We just need to use Storage abstraction instead of hard link to file storage.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
@rustamwin would you please check if the code works with the debugger?

**Comment by rustamwin** [comment]
> @rustamwin would you please check if the code works with the debugger?

I checked. `DebugController::index` doesn't work as expected. Returns only one record.

**Comment by samdark** [comment]
@xepozz would you please take a look?

**Pull request merged**